Neos.Neos/Classes/Domain/Service/NodeDuplication/Commands.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Neos\Domain\Service\NodeDuplication;
+
+use Neos\ContentRepository\Core\CommandHandler\CommandInterface;
+
+/**
+ * @implements \IteratorAggregate<CommandInterface>
+ */
+final readonly class Commands implements \IteratorAggregate, \Countable
```

Comment:
I don't really like this here, it seems confusing that there is no generic Commands collection class in the CR but then there is one here in Neos NodeDuplication. Anyone searching for somehting like this will trip over it...